src/internal/testutil/auth.go

```diff
@@ -43,6 +40,34 @@ func ActivateAuth(tb testing.TB) {
 	require.NoError(tb, err)
 }
 
+func ActivateAuthNewClient(tb testing.TB) {
```

Comment:
If the functions in this file are all new additions (so only stuff in this PR is calling them), I think we should avoid adding any helpers that know about the singleton cluster.

Any new authentication helpers here should take a pach client, something like
```AuthenticatePachClient(t testing.TB, c *client.APIClient, subject string) *client.APIClient```

The client passed by an argument remains unchanged and the newly returned pach client has the updated authentication.  That would be a true utility function, only adding value, and not costing anything in terms of side effects or resource allocation, which may or may not be desired.  It could work for any pachclient even if we didn't deploy on minikube.

Reply:
great point